Software/src/wizard/ZonePlacementPage.cpp

```diff
@@ -431,6 +442,13 @@ void ZonePlacementPage::onNumberOfLeds_valueChanged(int numOfLed)
 	checkZoneIssues();
 }
 
+void ZonePlacementPage::onNumberOfLeds_valueChanged(int numOfLed)
+{
+	Q_UNUSED(numOfLed);
+	using namespace std::chrono_literals;
+	_ledNumberUpdate.start(500ms);
```

Comment:
Could you add a comment why the delay is needed and why it is that large please?